src/Sentry.Unity.Tests/SimpleTests.cs

```diff
@@ -1,16 +1,26 @@
-﻿using NUnit.Framework;
+﻿using System.Collections;
+using NUnit.Framework;
 using UnityEngine;
+using UnityEngine.TestTools;
 
 namespace Sentry.Unity.Tests
 {
     public sealed class SimpleTests
     {
-
         [Test]
         public void SentryUnity_Object_NotNull()
         {
             var unitySentryOptions = ScriptableObject.CreateInstance<UnitySentryOptions>();
             Assert.NotNull(unitySentryOptions);
         }
+
+        [UnityTest]
+        public IEnumerator UnityCoroutine_CreatedGameObjectWithDelay_Found()
+        {
+            var cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            cube.name = "test cube";
+            yield return new WaitForSeconds(2);
```

Comment:
Is the test going to take 2 seconds to pass?
If so is there another way we can test this then? WaitForMilliseconds(1) or something?

Reply:
We can use `yield return null` to skip a frame, but there is no `WaitForMilliseconds`. It's just a test to show that coroutines work, we probably won't need it.

Reply:
I'd go with the fastest approach that still validates your test.

Reply:
Got you, will do `yield return null` then.